### PR TITLE
Ensure that cloudtrail check always uses S3 boto service

### PR DESCRIPTION
**Pull request by killthrush**
### Description

- [ ] Existing GitHub Issue? Please provide #: 

### Acceptance checklist

- [ ] Feedback addressed
- [ ] Tests added or modified appropriately
- [ ] All tests passed

### CODE REVIEW

File: reactor/features/accounts/facades/cloudtrail_bucket.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ def verify_read(aws, bucket_name, cloud_account_id):
         return ResourceAccess.NOT_OK, 'Bucket name was not supplied'
 
     try:
+        aws.service = 's3'
```
[the remark this answers is not in the document]
Comment (Tankanow) [reply reactor/features/accounts/facades/cloudtrail_bucket.py]: I'm confused how this fixes the issue. The first thing that happens in `_get_cloudtrail_config` is it sets `aws.service = 'cloudtrail'`.

[the remark this answers is not in the document]
Comment (imcgaunn) [reply reactor/features/accounts/facades/cloudtrail_bucket.py]: this solution makes sense, and the cause identified would explain the issue.

[the remark this answers is not in the document]
Comment (imcgaunn) [reply reactor/features/accounts/facades/cloudtrail_bucket.py]: wait. now I'm confused.

[the remark this answers is not in the document]
Comment (killthrush) [reply reactor/features/accounts/facades/cloudtrail_bucket.py]: The error is thrown on line 115 when we switch BACK to the previous service.  Which if it's not set explicitly somewhere is `None`

[the remark this answers is not in the document]
Comment (imcgaunn) [reply reactor/features/accounts/facades/cloudtrail_bucket.py]: okay. order is restored

**Pull request merged**